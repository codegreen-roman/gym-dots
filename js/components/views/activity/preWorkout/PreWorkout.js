import React from 'react'
import { ManageExerciseList } from './manageExerciseList/ManageExerciseList'

export const PreWorkout = () => {
    return (
        <section>
            <ManageExerciseList />
        </section>
    )
}
