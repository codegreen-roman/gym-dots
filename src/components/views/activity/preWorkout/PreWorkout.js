import React from 'react'
import { ManageExerciseList } from './manageExerciseList/ManageExerciseList'

// pre exercise
export const PreWorkout = () => {
    return (
        <section>
            <ManageExerciseList />
        </section>
    )
}
