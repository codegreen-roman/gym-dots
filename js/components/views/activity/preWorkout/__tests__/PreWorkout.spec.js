import React from 'react'
import { PreWorkout } from '../PreWorkout'
import { shallow } from 'enzyme'

const setup = () => {

    const props = {}

    return {
        component: shallow(<PreWorkout {...props} />),
        props
    }
}

describe('PreWorkout component', () => {
    it('should contain one ManageExerciseList component', () => {
        const { component } = setup()
        expect(component.find('Connect(_ManageExerciseList)').length).toBe(1)
    })

    it('should match the snapshot', () => {
        const { component } = setup()
        expect(component).toMatchSnapshot()
    })
})
