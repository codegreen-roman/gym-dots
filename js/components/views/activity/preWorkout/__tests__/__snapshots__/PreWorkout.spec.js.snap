// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreWorkout component should match the snapshot 1`] = `
<section>
  <Connect(_ManageExerciseList) />
</section>
`;
